doc/examples/plot_circular_elliptical_hough_transform.py

```diff
@@ -128,10 +128,10 @@
 
 # Estimated parameters for the ellipse
 best = result[-1]
```

Comment:
Ah, good points all. How about:

```python
best = list(result[-1])  # or `result.tolist()` if this doesn't work, but I find `list()` more readable
yc, xc, a, b = [int(round(x)) for x in best[1:5]]
```

(Since most Pythonistas aren't happy with lambdas...)